### PR TITLE
Update support for before/after print event handlers (Blink)

### PR DESCRIPTION
**Pull request by joseluisq**
This PR updates support for before/after print events.
According to https://www.chromestatus.com/feature/5700595042222080

**Chromium status**

- Chrome Desktop: v63+
- Chrome for Android: v63+
- Opera Desktop 50+

**Demo used:**
https://codepen.io/joseluisq/full/VzRpGb/

**References:**
- [MDN - WindowEventHandlers onafterprint browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onafterprint#Browser_compatibility)
- https://caniuse.com/#feat=beforeafterprint

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Also <https://www.chromestatus.com/feature/5700595042222080>: Opera 50+, too :)

**Comment by joseluisq** [comment]
> Also https://www.chromestatus.com/feature/5700595042222080: Opera 50+, too :)

true! 😄

**Comment by joseluisq** [comment]
Perhaps, if Opera 50+ support should come in separated PR. what do you think?

File: features-json/beforeafterprint.json [hunk 1]
```diff
@@ -287,6 +287,7 @@
       "37":"n"
     },
     "and_chr":{
+      "63":"y",
```
Comment (joseluisq) [inline features-json/beforeafterprint.json]: According to https://www.chromestatus.com/features/5700595042222080

**Comment by joseluisq** [comment]
@Schweinepriester What property correspond to **Opera for Android**, `op_mob` ?

**Comment by joseluisq** [comment]
> I don't think Opera for Android is tracked (anymore) really, but it probably was op_mob, yeah.
> 
> FYI the checks are currently failing because you added and_chr 63, but https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md#unsupported-changes
> 
>     Add a new browser or browser version

**Comment by joseluisq** [comment]
> I don't think Opera for Android is tracked (anymore) really, but it probably was op_mob, yeah.
> 
> FYI the checks are currently failing because you added and_chr 63, but https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md#unsupported-changes
> 
>     Add a new browser or browser version

Done.

**Comment by Fyrd** [comment]
Thanks both!

**Comment by ericpaul9978** [comment]
Stop

**Pull request merged**